crates/turbopack-ecmascript/src/references/esm/export.rs

```diff
@@ -147,6 +139,21 @@ impl CodeGenerateable for EsmExports {
                         );
                     }
                 }
+
+                if let EcmascriptExports::CommonJs = &*asset.get_exports().await? {
+                    let referenced_asset = esm_ref.get_referenced_asset().await?;
```

Comment:
Nit: we've already awaited on this value above, but we extracted `asset` out of the `ReferencedAsset::Some`. Can we fix this to await once?